autoprotocol/builders.py

```diff
@@ -1643,7 +1655,7 @@ def device_mode_params(
                 f"Incorrect params: {error_values}. It must be {accepted_params}"
             )
 
-        return {device: device_dict}
+        return accepted_params
```

Comment:
Do we need to return?

Reply:
nope! see reply to above comment. Thanks for pointing this out!